feature_engine/outliers/winsorizer.py

```diff
@@ -1,7 +1,7 @@
 # Authors: Soledad Galli <solegalli@protonmail.com>
 # License: BSD 3 clause
 
-from typing import List, Union
+from typing import List, Literal, Union
```

Comment:
I think we may need this change also in the OutlierTrimmer? Did you check?

Reply:
And we probably need to add `fold_` in both docstrings, here and outliertrimmer, so might be worth adding it as a common string in the _docstring module.